pywbem/cim_obj.py

```diff
@@ -3576,10 +3574,8 @@ def case(s):
 
         ret = []
 
-        # TODO: Ignore host for CIMObject format
-        # if self.host is not None and format not in ('cimobject'):
-        #     # The CIMObject format assumes there is no host component
-        if self.host is not None:
+        if self.host is not None and format not in ('cimobject'):
```

Comment:
Question. Is this the most efficient way to determine value of format?  I would not thought of this syntax

Reply:
No, it is not the most efficient format. I was just doing this for consistency with the other tests, but I will change it to a simple comparison.